src/utils/validation-annotations.ts

```diff
@@ -79,3 +79,33 @@ export function IsValidApiKey(validationOptions?: ValidationOptions) {
     });
   };
 }
+
+/**
+ * Validator constraint class for checking if a date (in form of string) is after or equal to the provided date.
```

Comment:
Requests are not working, nor with postman neither with Swagger. Please check this validation function

Reply:
Solved.